storage/aptosdb/src/state_store/mod.rs

```diff
@@ -1193,6 +1196,41 @@ impl StateValueWriter<StateKey, StateValue> for StateStore {
         self.ledger_db.metadata_db().put_usage(version, usage)
     }
 
+    fn kv_finish(&self, version: Version) -> Result<()> {
+        if let Some(internal_indexer_db) = self.internal_indexer_db.as_ref() {
+            let batch = SchemaBatch::new();
+            if version > 0 {
```

Comment:
you sure we don't write down a `None` explicitly?

Reply:
Yes we don't need to write it.